### PR TITLE
Add shitty logs and code to reproduce artifacts

### PR DESCRIPTION
**Pull request by ggarber**

### CODE REVIEW

File: worker/src/RTC/Producer.cpp [hunk 1]
```diff
@@ -657,10 +663,17 @@ namespace RTC
 			MS_ABORT("found stream does not match received packet");
 		}
 
+
+
+				static int counter = 0;
+		if (packet->GetSsrc() != rtpStream->GetRtxSsrc() && counter++ % 8 == 0)
```
Comment (ggarber) [inline worker/src/RTC/Producer.cpp:669]: discard 1/8th of the packets to simulate packet loss (don't discard rtx packets)

File: worker/src/RTC/SimulcastConsumer.cpp [hunk 2]
```diff
@@ -569,13 +569,31 @@ namespace RTC
 		MS_ASSERT(this->externallyManagedBitrate, "bitrate is not externally managed");
 		MS_ASSERT(IsActive(), "should be active");
 
+		// ggb
+		static int counter = 0;
+		if (counter++ % 2 == 0)
```
Comment (ggarber) [inline worker/src/RTC/SimulcastConsumer.cpp:574]: switch spatial layer every second

**Comment by ibc** [comment]
I mean... we cannot merge this obviously

**Comment by ibc** [comment]
Closing because obviously... welll XD

**Comment by ggarber** [comment]
LOL I'm not aware of having made a PR for this branch :).  Maybe my computer is infected like catalan's politicians phones :)

Anyway that gives you a clue of one issue we have and are trying to debug 

Apologize for the inconvenience

**Pull request closed without merging**